src/repository.h

```diff
@@ -79,5 +79,7 @@ PyObject* Repository_blame(Repository *self, PyObject *args, PyObject *kwds);
 PyObject* Repository_merge(Repository *self, PyObject *py_oid);
 PyObject* Repository_cherrypick(Repository *self, PyObject *py_oid);
 PyObject* Repository_apply(Repository *self, PyObject *py_diff);
+PyObject* Repository_merge_analysis(Repository *self, PyObject *py_id);
```

Comment:
Couldn't see why `Repository_merge_analysis` was missing here already (oversight?) so I added it.